### PR TITLE
Created a To Do List structure

### PR DESCRIPTION
**Pull request by dazehinn**
### Features
* An array of tasks
* use of webpack
* activities populated dynamically on window load

### CODE REVIEW

**Review by Mihndim2020 — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @dazehinn,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure your implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._
- [ ] You have submitted a project that does not have linters correctly set up. Please checkout this [link ](https://github.com/microverseinc/linters-config/tree/master/javascript)for directives. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **does not count** into the code reviews limit._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "html-webpack-plugin": "^5.5.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (mikethreels) [inline package.json]: - [ ] according to the requirements Webpack dev server should be set up and running with no errors.

File: README.md [hunk 2]
```diff
@@ -1 +1,159 @@
...
+- git (for version control)
+- web browser
+Have text editor installed. (VSCode, sublime, atom)
+- (optional) Have CSS,HTML & JS Linters installed and configured. For further information, visit [this link](https://github.com/microverseinc/linters-config/blob/master/README.md) to get better understanding on each linters used on this project.
+
+### Setup
+
+* Clone this repository to your desired folder
+* open the index.html file using a web browser
+
```
Comment (mikethreels) [inline README.md]: - [ ] when following the setup instructions the page shown is just a blank screen with the text "This is a ToDo app"

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Nshanji, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @dazehinn 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Descriptive PR description :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**